src/main/java/frc/robot/commands/CompositeCommands.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.notepath.Notepath;
+import frc.robot.subsystems.shooter.ShooterBed;
+import frc.robot.subsystems.shooter.ShooterFlywheel;
+
+public final class CompositeCommands
+{
+    private CompositeCommands()
+    {
+    }
+
+    public static Command intakePickup(Intake intake, Notepath notepath)
+    {
+        return Commands.parallel(IntakeCommands.startIntake(intake), NotepathCommands.intakePickup(notepath)).andThen(Commands.waitUntil(() -> notepath.hasNote())).finallyDo(() ->
+        {
+            intake.setIntakeOff();
+            notepath.setOff();
+        });
```

Comment:
This command looks fine, but we only want to be able to pick up a note if we don't already have one. I'm thinking we need to add something like this:

```suggestion
        return Commands.parallel(IntakeCommands.startIntake(intake), NotepathCommands.intakePickup(notepath)).andThen(Commands.waitUntil(() -> notepath.hasNote())).finallyDo(() ->
        {
            intake.setIntakeOff();
            notepath.setOff();
        }).unless(() -> notepath.hasNote());
```